### DISCUSSION TITLE
how can create relationships with other contentTypes?

### DISCUSSION THREAD

**Discussion by adrian-ub** [estruyf/vscode-front-matter#583] [Q&A]
Relationship like Netlify CMS

https://decapcms.org/docs/widgets/relation/

**Comment by estruyf** [comment]
Can you give me more context on what you want to achieve? Do you wish to reuse fields from another content type?

**Reply by adrian-ub** [reply]
example:

we have two contentTypes "authors" and "posts", from posts a list appears with the authors that exist

It's something like "data.files", but not in a json/yml file, but both contentTypes in markdown files

**Reply by estruyf** [reply]
Ah, so you got an `authors` folder with, for instance: `author 1` en `author 2` in it. When creating a post, `author 1` and `author 2` should appear in the relationship field. Is that right?

How should the relationship get established? Is it the path to the author md file?

**Reply by adrian-ub** [reply]
Yes, it can be with the path of the md file or the slug

authors
```md
---
title: 'Adrián UB'
slug: adrianub
---
```

posts:

```md
---
title: 'hello world'
authors:
  - adrianub
---
```

or

```md
---
title: 'hello world'
authors:
  - authors/adrianub.md
---
```

**Reply by estruyf** [reply]
Thank you @adrian-ub, this would be a good new field type. 🙏

**Reply by estruyf** [reply]
Added it to the backlog: https://github.com/estruyf/vscode-front-matter/issues/585
